lib/internal/cluster/master.js

```diff
@@ -41,15 +42,15 @@ if (schedulingPolicy === undefined) {
 
 cluster.schedulingPolicy = schedulingPolicy;
 
-cluster.setupMaster = function(options) {
+cluster.setupMaster = function(options = {}) {
```

Comment:
@jasnell Can you explain why? It’s not obvious to me how this would break code that isn’t designed specifically to be broken by this change.